changelog/9085.improvement.md

```diff
@@ -0,0 +1,2 @@
+Add support for `cafile` parameter in `endpoints.yaml`.
+This will load a custom local certificate file and use it when making requests to that endpoint.
```

Comment:
can we please add an example for the `endpoints.yml` here? I couldn't find where this is covered in the docs and also reached out the CSE [here](https://rasa-hq.slack.com/archives/C016FLYV4QN/p1626192744055800) because I think this is vital

Reply:
Thanks for reaching out, I was also having the same trouble finding a suitable location. The closest I could find was `docs/docs/docker/building-in-docker.mdx`. Although this doesn't seem ideal. 
I will add an example in the changelog like you suggest, and we can see what CSE thinks wrt the docs.